bin/live/pycbc_live_combine_single_fits

```diff
@@ -154,28 +154,32 @@ for ifo in args.ifos:
     logging.info(ifo)
     fout_ifo = fout.create_group(ifo)
     l_times = np.array(live_times[ifo])
-    fout_ifo.attrs['live_time'] = l_times.sum()
+    total_time = l_times.sum()
+    fout_ifo.attrs['live_time'] = total_time
 
     fout_ifo['separate_fits/live_times'] = l_times[ad_order]
     fout_ifo['separate_fits/start_time'] = trigger_file_starts[ad_order]
     fout_ifo['separate_fits/end_time'] = trigger_file_ends[ad_order]
 
-    for counter, a_c_u_l in enumerate(zip(alphas_bin[ifo],
-                                          counts_bin[ifo], bu, bl)):
-        a, c, u, l = a_c_u_l
+    for counter, (a, c) in enumerate(zip(alphas_bin[ifo],
+                                         counts_bin[ifo])):
         # Sort alpha and counts by date
         a = np.array(a)[ad_order]
         c = np.array(c)[ad_order]
-        invalphan = c / a
-        mean_alpha = c.mean() / invalphan.mean()
-        cons_alpha = np.percentile(a, 100 - args.conservative_percentile)
+
+        # ignore anything with the 'invalid' salient values
+        valid = c > 0
```

Comment:
Hmm, what happens if all items of `valid`are False? Presumably the lines below will crash?

Reply:
This would be the case when the interferometer had unusable data for ~20 days, which is certainly unlikely but maybe we should decide upfront what we want the code to do in that situation?

Reply:
I've added in a 'if not any(valid)' check to use nan values if that is the case.

I'll need to update what happens in the case that the search itself gets a nan from this file though for that edge case

Reply:
Added now